web/html/src/utils/network.js

```diff
@@ -51,8 +51,10 @@ function errorMessageByStatus(status) {
         return [t("Authorization error, please reload the page or try to logout/login again.")];
     } else if (status >= 500) {
         return [t("Server error, please check log files.")];
+    } else if (status === 400) {
```

Comment:
Sorry for being picky here :smile: I think the errors between 400 and 499 are **client** errors, not server ones. Moreover, I think that differentiating message for this one would be good  (e.g. report "Client error" for 400 and "server error for >= 500").

Reply:
well yeah, but for the customer everything is a server error, IMO that is some kind of internal definition, for them, I would say it's hard to distinguish what is the difference between a client/server error. Anyway, I removed that and kept the old behavior, most of the cases we want to have custom error handling for 400 and show something like "Check all the input fields errors bellow"